### ISSUE TITLE
Nodes overlap on wide trees

### ISSUE THREAD

**Issue by pconnell99** [fperucic/treant-js#90]
In a project I'm working on, I am getting nodes that are overlapping when there are a lot of siblings on the same level.  I am unsure if this is Treant or my custom CSS width for nodes.

Example attached.  Note the clash on the second tier, in the middle.

![chart](https://user-images.githubusercontent.com/11887266/32547613-e4dd0754-c47a-11e7-9d51-7b79815c660d.png)

My css for a node is:-

```
.node {
  color: #000000;
  border: 2px solid #000000;
  border-radius: 8px;
  width: 175px;
  font-family: Arial;
}
```

Is this me or Treant?  Fixable?

**Comment by fperucic** [comment]
I think it is a problem with Treant and `stackChildNodes` property

**Comment by pconnell99** [comment]
So if I switched off `stackChildNodes` for now, that could be a temporary fix?

**Comment by dlgoodchild** [comment]
Yeah it looks like it, it's most likely not taking into account the horizontal offset when being stacked which affects the true width of the nodes used in calculating their positions.

**Comment by pconnell99** [comment]
Can you point me in a rough direction in the lib and I could take a look.

**Comment by dlgoodchild** [comment]
My immediate guess is around line 583 (`// handle stacked children positioning`).
It used the stackIndent, but I suspect there's a miscalculation in and around that area.

**Comment by pconnell99** [comment]
Not having much luck here.  Any more pointers?
